### PR TITLE
Blog post with SWC/DC merger update

### PR DESCRIPTION
**Pull request by raynamharris**

### CODE REVIEW

File: _posts/2017/06/2017-06-17-merger.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ 
+Below is a timeline for the merger process. Various steps in this timeline require approval by the two governing boards, and are dependent on results of previous steps, so this may change as we proceed. 
+ 
+| Month | Merger Process Milestones | 
+| --- | --- | 
+|June | Communicate the in-person meeting outcomes and proposed governance structure to staff and community members. | 
+|July | Present motions to approve the proposed structure of the governing board, executive leadership, and transfer of assets (e.g. finances, cross-organization subcommittees). Enumerate required bylaws.|
+|August | Present and approve an official motion to combine Software Carpentry and Data Carpentry into a unified organization. Approve nominations for the appointed seats to the board.|
+|September | Approve a unified budget. Approve the transfer of assets. |
+|October | Call for candidates for elected members of the board. Approve motions to dissolve the current governing boards and transfer assets as of January 1. |
```
Comment (rgaiacs) [inline _posts/2017/06/2017-06-17-merger.md]: Any specific reason for January 1? Financial year or something?

**Comment by kcranston** [comment]
Note that @tracykteal requests that we use 'Steering committee' rather than 'Board of governors' throughout due to fiscal sponsorship rules. See https://github.com/datacarpentry/datacarpentry.github.io/pull/243

File: _posts/2017/06/2017-06-17-merger.md (hunk 1 above)
Comment (kcranston) [reply _posts/2017/06/2017-06-17-merger.md]: We wanted to pick a date, and given the timeline, it was looking like end of Dec / beginning of Jan would work, so Jan 1 seemed clean and tidy.

**Comment by raynamharris** [comment]
All the terminology has been updated. Good to go now.

**Pull request merged**